packages/test/end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -79,7 +80,7 @@ const tests = (args: ILocalTestObjectProvider) => {
     });
 
     beforeEach(async () => {
-        request = args.urlResolver.createCreateNewRequest(documentId);
+        request = createLocalResolverCreateNewRequest(documentId);
```

Comment:
>createLocalResolverCreateNewRequest [](start = 18, length = 35)

should this be route thru the ITestObjectProvider?

Reply:
these tests are still coupled to the local server, as with some others. i'll be fixing that is a follow up. the lines between each PR are a bit blurry but trying to break it apart as reasonably as possible.